Resources/Textures/Structures/Wallmounts/camera.rsi/meta.json

```diff
@@ -1 +1,88 @@
\ No newline at end of file
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from /tg/station at https://github.com/tgstation/tgstation/commit/6bfe3b2e4fcbcdac9159dc4f0327a82ddf05ba7bi",
+    "states": [
+        {
+            "name": "camera",
+            "directions": 1,
+            "delays": [
+                [ 0.3, 0.3, 0.3, 0.3, 0.3, 0.3, 0.3 ]
+            ]
+        },
+        {
+            "name": "camera_assembly",
+            "directions": 1,
+            "delays": [
+                [ 1.0 ]
+            ]
```

Comment:
1.0 delays aren't needed and can be pruned.

Reply:
Good to know! Pruned.